index.html

```diff
@@ -38,14 +38,23 @@
   </head>
   <body>
     <section  id="abstract">
-      <p> This document describes architecture on Web of Things which derived from use cases.<br />
-      The architecture consists of building blocks and the function of each block is explained.<br />
-      And it is also described how to map the WoT architecture to the wide variety of real implementation patterns.<br />
-      <br />
-      This document focused on the standardization scope of Web of Things.<br />
-      And other blocks and conditions underlying the Web of Things are described as example implementation and/or requirements such as the description of how to treat constraint devices and security and privacy.<br />
-      Web of Things is distributed platform over the latest Web technology. And Web of Things can wrap the difference among legacy IoT protocols defined by various standardizing organizations and also can provide interoperability between above legacy standards.<br />
-      Utilizing this interoperability, IoT application developers can ship only one wot-compliant application on behalf of shipping each legacy standard-compliant application.</p>
+      <p> This document describes an architecture for the Web of Things.
+      This architecture is derived from a set of use cases and consists of
+      a set of building blocks.
+      Each building block in the architecture is defined and its function explained.

```

Comment:
My current revision adds a bit of explanation to the main deliverables in the abstract and explains how they fit in.   Note that I emphasize the primality of the TD and the secondary (and optional) nature of the Scripting API (and so the WoT runtime).   I've tried to avoid the term "platform" and talk about devices "participating" in the WoT instead (which I think is a better way to thing about it).